content/01.abstract.md

```diff
@@ -2,6 +2,13 @@
 
 Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
 However, traditional research and multi-author writing processes break down at scale.
-New tools and workflows that rely on automation can catch errors and ensure that contributions are fairly tracked in massively collaborative research.
-We present techniques for overcoming challenges of open research, with special emphasis on manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and many other aspects of manuscript building.
+We present a new software tool named Manubot, available at <https://manubot.org>, for open scholarly writing.
+Manubot adopts the contribution workflow used by many large-scale open source software projects to enable collaborative authoring of scholarly manuscripts.
+With Manubot, manuscripts are written in Markdown and stored in a Git repository to precisely track changes over time.
+By hosting manuscript repositories publicly, such as on GitHub, multiple authors can simultaneously propose and review changes.
+Publication with Manubot is continuous.
```

Comment:
I'm not sure that `continuous` is a common enough idea to stand on its own here.  Could we move this to be part of the next sentence?  Something like?
> When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page, providing continuous publication.

Reply:
How about switching to a colon:

```
Publication with Manubot is continuous:
When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
```

I like the flow of having the clauses separate, and the colon should make the explanatory nature of the second sentence more clear.

Capitalization style info for this situation at https://www.grammarly.com/blog/capitalization-after-colons/

Reply:
This is really nitpicky, but wouldn't a semicolon be the appropriate punctuation in that scenario?

Reply:
From https://writingcenter.unc.edu/tips-and-tools/semi-colons-colons-and-dashes/

> Colons follow independent clauses (clauses that could stand alone as sentences) and can be used to present an explanation, draw attention to something, or join ideas together.

Reply:
A colon follows one independent clause, but a semicolon is better between two independent clauses.  Either way, the punctuation makes for a complex sentence in the abstract.

Can we keep separate sentences but make it active:
> Manubot provides continuous publication.
When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.

A downside is that many sentences will start with `Manubot ...`

Reply:
I think we might be overthinking it here. The sentence as written with the colon looks totally fine and normal to me. Regardless of what the official english/websters dictionary grammar specifies, I've seen sentences written like that with a colon all the time.

Reply:
> A colon follows one independent clause, but a semicolon is better between two independent clauses.

@agitter I think you may not be aware of this usage of a colon where the colon indicates that the second independent clause explains the first. See the [grammergirl examples](https://www.grammarly.com/blog/capitalization-after-colons/):

> It’s been snowing for three days straight: The roads around here aren’t very safe for driving.

> Maggie wears a brimmed cap at all times: Strong light often gives her a headache. She also likes the way it looks.

> Either way, the punctuation makes for a complex sentence in the abstract.

I like the colon because I think it maintains most of the separation and succinctness of the current construction. In other words, I don't really view the colon as creating a complex sentence. To the contrary, I think it helps retain simplicity while indicating the second sentence explains continuous publication.

Regarding "Manubot provides continuous publication.", I think it's less persuasive with a worse flow, which is why I'm being so insistent.

I do think the suggesting below is a decent alternative:

> A cloud service automatically evaluates proposed changes to catch errors and provide continuous publishing.

However, I think it's a bit more jargon-heavy. One goal with the current construct is to not lose readers who don't understand what  "Continuous Publication" means.

Reply:
One final thing is that we should think about whether to use the term "continuous publishing" or "continuous publication". There are a few instances in the manuscript, and we should make them all consistent.

From https://wikidiff.com/publishing/publication:

> As nouns the difference between publishing and publication is that publishing is the industry of publishing, including the production and distribution of books, magazines, web sites, newspapers, etc while publication is the act of publishing printed or other matter. 

http://blog.martinfenner.org/2014/03/10/continuous-publishing/

Reply:
I like publishing. 👍

Reply:
continuous publishing has [674 Google Scholar results](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C39&q=%22continuous+publishing%22&btnG=) versus [4,400](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C39&q=%22continuous+publication%22&btnG=) for "continuous publication".

However, much of the usage of "continuous publication" is regarding journals moving away from issue-based publishing. [Example](https://doi.org/10.1259/bjr.20130252)

> Continuous publication focuses on publishing individual articles as opposed to the more traditional issue-based publishing. Once an article is in its final form, after editing and proofreading, it is immediately published online as the version of record straight into an issue. There is no delay caused by waiting for a group of manuscripts to be gathered together for a given issue. This drip feeding of articles means that issues are built up gradually throughout the month.

Continuous publishing is used by Fenner's [post](http://blog.martinfenner.org/2014/03/10/continuous-publishing/) to describe Manubot's use case.

Hence, I feel like we're slightly coining the term, so we should probably just go with what we think is most accurate?

Reply:
@dhimmel I'll defer to you regarding the punctuation and the publishing / publication decision.  Whatever you find to be more accurate works for me.

Reply:
Will address in https://github.com/greenelab/meta-review/pull/216